app/views/map/_inlineLeaflet.html.erb

```diff
@@ -5,24 +5,40 @@
 
   <style>
     #map<%= unique_id %> { width:100%; height:300px; margin: 0; position: relative;}
+    #postOnMap<%= unique_id %> {
+      margin-top: 5px;
+    }
   </style>
-  <div class="leaflet-map" id="map<%= unique_id %>"></div>
 
+  <div class="leaflet-map" id="map<%= unique_id %>"></div>
+  
+  <p><i><small>
+    <button id="postOnMap<%= unique_id %>" class="btn btn-primary">Add a story +</button>
+  </small></i></p>
 
 <script>
     var bounds = new L.LatLngBounds(new L.LatLng(84.67351257 , -172.96875) , new L.LatLng(-54.36775852 , 178.59375)) ;
 
-
     var map<%= unique_id %> = L.map('map<%= unique_id %>' , {
         maxBounds: bounds ,
         maxBoundsViscosity: 0.75
     }).setView([<%= lat %>,<%= lon %>], <%= lat.to_s.length.to_i %> + 6);
 
+    var button<%= unique_id %> = document.getElementById('postOnMap<%= unique_id %>');
+
+    button<%= unique_id %>.addEventListener('click', function(event) {
+      var center_lat = map<%= unique_id %>.getCenter().lat;
+      var center_lng = map<%= unique_id %>.getCenter().lng;
+      window.location.href='/post?tags=lat:' + center_lat + ',lon:' + center_lng + '&title=This post will apppear on maps';
```

Comment:
Hey @jywarren ,

Changed the CSS : 

<img width="857" alt="Screenshot 2019-11-02 at 1 12 53 AM" src="https://user-images.githubusercontent.com/14952645/68052288-53932600-fd0f-11e9-8f7e-52d0e96bb261.png">

This line opens the /post form with tags lat:XX and lon:XX 😄 . I have added the next steps in the PR description --- do check them please.

The tags appear as (calculated from map's center) : 

<img width="871" alt="Screenshot 2019-11-02 at 1 29 34 AM" src="https://user-images.githubusercontent.com/14952645/68052732-504c6a00-fd10-11e9-99e2-955a445d7164.png">